README.md

```diff
@@ -1,176 +1,203 @@
-│   ├── dags
-│       ├── test_dag_integrity.py
-├── Dockerfile
-├── include
-├── packages.txt
-├── plugins
-└── requirements.txt
+To install a specific version of the CLI, specify the version number as a flag at the end of the command. For example, to install v1.1.0 of the CLI, you would run:
+
+```sh
+curl -sSL install.astronomer.io | sudo bash -s -- v1.1.0
 ```
 
-DAGs can go in the `dags` folder, custom Airflow plugins in `plugins`, python packages needed can go in `requirements.txt`, and OS level packages can go in `packages.txt`.
+:::info
 
-1. Start airflow
+If you receive a `mkdir` error during installation, download and run the [godownloader](https://raw.githubusercontent.com/astronomer/astro-cli/main/godownloader.sh) script locally.
 
-Run `astro dev start` to start a local version of airflow on your machine. This will spin up a few locally running docker containers - one for the airflow scheduler, one for the webserver, and one for postgres.
-(Run `astro dev ps` to verify)
+    $ cat godownloader.sh | bash -s -- -b /usr/local/bin
 
-## Help
+:::
 
-The CLI includes a help command, descriptions, as well as usage info for subcommands.
 
-To see the help overview:
+#### Troubleshoot installation issues
 
-```
-$ astro help
-```
+If you encounter issues when installing the Astro CLI:
 
-Or for subcommands:
+- Make sure Docker Desktop is installed correctly. See [Install Docker Desktop on Windows]https://docs.docker.com/desktop/install/windows-install/).
 
-```
-$ astro dev --help
-```
+- Make sure the Docker Desktop WSL 2 backend is installed and configured correctly. See [Docker Desktop WSL 2 backend](https://docs.docker.com/desktop/windows/wsl/).
 
-```
-$ astro deploy --help
-```
+- Enable Hyper-V on the Docker and Linux Containers and run the Astro CLI natively. For additional information, see the [Docker docs troubleshooting guide](https://docs.docker.com/desktop/troubleshoot/overview/).
+
+## Get started
+
+1. Create a project
+
+    ```
+    $ mkdir hello-astro && cd hello-astro
+    $ astro dev init
+    ```
+
+2. Install the binary and Confirm the install worked:
+
+    ```
+    $ ./astro
+    ```
+
+    This generates a basic project directory:
+
+    ```
+    .
+    ├── dags
+    │   ├── example_dag_advanced.py
+    │   ├── example_dag_basic.py
+    ├── tests
+    │   ├── dags
+    │       ├── test_dag_integrity.py
+    ├── Dockerfile
+    ├── include
+    ├── packages.txt
+    ├── plugins
+    └── requirements.txt
+    ```
+
+    DAGs can go in the `dags` folder, custom Airflow plugins in `plugins`, python packages needed can go in `requirements.txt`, and OS level packages can go in `packages.txt`.
 
-## Local Development
+3. Run `astro dev start` to start a local version of airflow on your machine. This will spin up a few locally running docker containers - one for the airflow scheduler, one for the webserver, and one for postgres.
+(Run `astro dev ps` to verify).
 
-How to get started as a developer:
+## Local development
```

Comment:
It would be great if we could remove everything about local development to a new `CONTRIBUTING.md` doc at the top level of the repo. This content is primarily for Astronomer developers, and I don't want people to mistake it for "Airflow project development"

Reply:
If we are doing this then we should also move the "Run Tests" & "Generate Mocks" sections to `CONTRIBUTING.md` as well

Reply:
@jwitz and @neel-astro I have created the recommended CONTRIBUTING.md doc. I'm not sure how to configure it properly, or make sure it appears "at the top level of the repo". I'll leave it to you to make the necessary adjustments.